client/lib/cart/store/index.js

```diff
@@ -151,6 +165,47 @@ CartStore.dispatchToken = Dispatcher.register( payload => {
 		case CART_ITEM_REPLACE:
 			update( cartItems.replaceItem( action.oldItem, action.newItem ) );
 			break;
+
+		case TRANSACTION_NEW_CREDIT_CARD_DETAILS_SET:
+			{
+				// typically set one or zero
+				const countryCode = get( action, 'rawDetails.country' );
+				const postalCode = get( action, 'rawDetails.postal-code' );
+				postalCode && update( setTaxPostalCode( postalCode ) );
+				countryCode && update( setTaxCountryCode( countryCode ) );
+			}
+			break;
+
+		case TRANSACTION_PAYMENT_SET:
+			{
+				// set both
+				let postalCode, countryCode;
+
+				const paymentMethod = get( action, [ 'payment', 'paymentMethod' ] );
+				switch ( paymentMethod ) {
+					case 'WPCOM_Billing_MoneyPress_Stored':
```

Comment:
Not a blocker, but worth making the payment methods names constants at some point

Reply:
Yeah, that's a good idea. It fits nicely into the other refactor I want to do too.

The problem I ran into was that I need a place to put stuff - we can't have circular dependencies from the stores requiring each other.

The shopping cart has `cart-values` for all the the operations on the store, so I created `transactions-values` and started putting stuff there and the diff started growing really fast, but that's the perfect place to put these constants.